solr_init.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Probably best to use a more portable shebang, `#!/usr/bin/env bash`
https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang